pup-0006.md

```diff
@@ -0,0 +1,99 @@
+created: 15-08-2018
+status: Approved
+version: 1
+---
+
+## Summary
+
+For [Pulp project repositories](https://github.com/pulp/), there is no written process to describe
+giving the commit bit to a contributor. The existing committers want to openly document the process.
+
+
+## Motivation
+
+Comitter - is an individual who has write permissions to a
+[Pulp Organization](https://github.com/pulp/) repository.
+To have "a commit bit" means that one is permitted to commit those changes. 
+
+Historically the commit bit was given on day 0 when hired on the core team at Red Hat.  In Oct 2017
+the existing committers decided to stop that practice and instead document an open process.
+Engineers hired on the Pulp team since Oct '17 have not received commit bit.
+
+We have not yet documented an open process of which to become a committer.
+The process outlined in this document is inspired by processes in Chromium projects and Apache
+Software Foundation.
+
+
+## Scope
+
+This policy is intended as the default for all software in the
+[Pulp Organization](https://github.com/pulp/) unless explicitly stated otherwise.
+
+
+## Becoming a committer.
+
+Overall the candidate must have demonstrated:
+* project commitment, active involvement and care towards the needs of all Pulp users and not only
+  their own interests
+* to provide constructive code reviews
+* the ability to collaborate with the team and understand its policies and workflows
+* understanding of the projects' code base and functionality by submitting code that was accepted
+  and merged in the repositories of interest
+
+These requirements are somewhat vague by design and leave the decision to the judgement of the
+existing committers. Users rely on the judgement of committers already, so to rely on that judgement
+to accept new contributors is reasonable. Anyone who specifically wants to get more involved should
+approach the existing commit bit holders about mentorship.
+
+
+## Process
+
+A nomination or self-nomination email should be sent to the pulp-dev email list and must contain
+the following:
+
+* Name and email of the nominee. This can be a self-nomination.
+* Explanation of why the nominee should be a committer.
+* Repository being requested.
+* Vote end date. Must be 7 calendar days from now.
+
+To pass, a +1 vote is required from the majority of existing committers for that repository and
+no -1 votes.
+
+If a nomination does not pass, the candidate may be re-nominated after a cool down period of 60
+days.
+
+
+# Revocation of the committer status
+
+If there are problems or disagreements within the community, they can usually be solved through
+open discussion and debate.
+
+In the unhappy event that a committer continues to disregard good citizenship (or actively disrupts
+the project), that person's status may be revoked. The process is similar as for nominating a new
+committer with a +1 voting of supermajority of the existing committers for that repository.
```

Comment:
We need to specifically say "two-thirds of the existing ...". The supermajority term can ambiguously mean three-fifths, 55percent, etc according to wikipedia. I mean it to be two-thirds.

Reply:
https://en.wikipedia.org/wiki/Supermajority

Reply:
Ah, this is a good clarification.